lib/registration/disparity_registration_simple.cpp

```diff
@@ -0,0 +1,81 @@
+/// \file
+/// Maintainer: Felice Serena
+///
+
+#include "disparity_registration_simple.h"
+#include <boost/log/trivial.hpp>
+
+namespace MouseTrack {
+
+PointCloud DisparityRegistrationSimple::
+operator()(const FrameWindow &window) const {
```

Comment:
Think about convention? Where to put reference: maybe as default on parent? Or does the parent provide a "frame"? set up stuff ...